src/components/Hero/Hero.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import HeroImage from "../../assets/hero-img.jpg"
+import "./Hero.scss"
+
+export default function Hero() {
+    return(
+        <div className="hero">
+            <div className="hero-image" style={{ backgroundImage: `url(${HeroImage})` }}></div>
```

Comment:
Hi Natalia,
for tags with empty body it is possible to use self-closing jsx tags, like `<div />`.
https://react-cn.github.io/react/tips/self-closing-tag.html

Reply:
@inductio, thanks, done.